### DISCUSSION TITLE
How to see all highlights after changing computer name

### DISCUSSION THREAD

**Discussion by ghost** [ahrm/sioyek#703] [Q&A] [answered]
When doing `gH` I do not see my highlights. I have restored all the config files and the .local/share data in a fresh operating system. I have changed the username so maybe that is the issue.

When opening a file I had previously highlighted I can see the highlights I made and they are added when I press `gH`. 

What I want is after doing `gH` to see all my highlights because it is a pain to open again all the pdfs I had highlighted one by one.

**Comment by ahrm** [comment, marked as answer]
It is not possible since the paths of all files are different now.

**Reply by ghost** [reply]
And could anything be done in the future for this, like the possibility in the configs to change the data location path/s?

**Reply by ahrm** [reply]
Maybe. For now, if you are savvy you could edit the `local.db` file (it is a simple sqlite database file) to update the file names to the correct ones.

**Reply by ghost** [reply]
Thanks. Sadly I am not savvy...
